Source/ChocolateyGui/App.xaml.cs

```diff
@@ -21,7 +22,20 @@ public partial class App
     {
         static App()
         {
-            Container = AutoFacConfiguration.RegisterAutoFac();
+            Container = AutoFacConfiguration.InitialRegistration();
+
+            var configurationProvider = Container.Resolve<IChocolateyConfigurationProvider>();

```

Comment:
That's exactly what I'm saying. Something like:
https://gist.github.com/RichiCoder1/a1e6c049777aca7c2093
(Note: haven't actually compiled that)